styles/styles.css

```diff
@@ -163,35 +153,45 @@ body::-webkit-scrollbar-thumb{
 }
 
 .work-image{
-    background-color: #ff7315;
-    border: 25px solid transparent;
-    border-image: url('../images/border.png') 115 stretch;
-    box-shadow: 5px 10px #232020;
-    margin: 0 2rem 4rem 2rem;
     position: relative;
-    background-position: center;
-    background-repeat: initial;
-    background-size: cover;
+}
+
+.work-image img{
+    border: 20px solid transparent;
+    border-image: url('../images/border.png') 140 stretch;
+    box-shadow: 5px 10px #232020;
     transition: all 0.3s ease;
+    width: 100%;
     height: 100%;
+    position: relative;
+    z-index: 0;
 }
 
 .work-text-container{
-    display: none;
     position: absolute;
-    opacity: 80%;
+    opacity: 0%;
     text-align: center;
-    background-color: #f4f4f4;
-    top: 10%;
+    background-color: #ff7315;
     height: 100%;
     width: 100%;
+    z-index: 1;
 }
 
 .work-image:active .work-text-container, .work-image:hover .work-text-container{
-    display: block;
+    opacity: 80%;
+    transition: all 0.3s ease;
+}
+
+.project-title{
+    font-family: 'oxanium', sans-serif;
+    font-size: 2rem;
+    position: relative;
+    top: 10%;
+    text-align: center;
```

Comment:
Don't think this does anything, it's already inheriting text center from it's parent